use-case-guides/workday-request-time-off/README.md

```diff
@@ -3,129 +3,96 @@ design_pattern_id: 2
 
 ## Discovery Questions to Personalize Your Conversation  Design *(Customization Options)*
 
-1. Which time off plans should be shown to employees in a given country (that you are launching to)?
-2. Do you specify time off in days or hours for these plans?
+1. Which time off plans should be shown to employees in a given team or given country (that you are launching to)?
+    * This determines the expected list of time off plans that will be shown to employees in the conversation design, as well as launch rules for your plugin.
```

Comment:
Style guide rule: if you're writing a compound sentence like this, it's better to list benefits / implications as separate bullet points

Reply:
formatted discovery questions to follow this format